service/src/main/java/uk/gov/hmcts/reform/fpl/enums/YesNo.java

```diff
@@ -2,10 +2,14 @@
 
 import lombok.Getter;
 
+import java.util.stream.Stream;
+
 @Getter
 public enum YesNo {
     YES("Yes"),
-    NO("No");
+    NO("No"),
```

Comment:
hmm the name for this class doesn't really make sense to me anymore with these new fields

Reply:
Yeah but if we make a change it has to be changed in lot many classes. :(

Reply:
We are planning to update the name to AnswerOptions but it will affect 17 other files. Will plan to do it in a separate pr